src/utils/validate.js

```diff
@@ -0,0 +1,74 @@
+import { MESSAGE } from '../constants/message.js'
+
+class validate {
+	inputMoney(inputMoney) {
+		this.#isNotNumber(inputMoney)
+		this.#isZeroOrminus(inputMoney)
+		this.#isNotDivision(inputMoney)
+	}
+	inputWinner(splitNumber) {
+		this.#isNotLengthSix(splitNumber)
+		this.#isNotOverLap(splitNumber)
+		this.#isNotNumberArr(splitNumber)
+	}
+	inputBonus(inputBonusNumber) {
+		this.#isNotNumber(inputBonusNumber)
+		this.#isLottoRange(inputBonusNumber)
+	}
+	randomLottoArr(randomNumber) {
+		this.#isLottoOverlap(randomNumber)
+		this.#isNotLengthSixLotto(randomNumber)
+	}
+
+	#isNotNumber(inputMoney) {
+		if (isNaN(inputMoney)) {
+			throw new Error(MESSAGE.ERROR.NOT_NUMBER)
+		}
+	}
+	#isNotNumberArr(splitNumber) {
+		for (let number of splitNumber) {
+			if (number.trim() === '') {
+				throw new Error(MESSAGE.ERROR.NOT_VALUE)
+			}
+		}
+	}
+	#isZeroOrminus(inputMoney) {
+		if (Number(inputMoney) === 0 || Number(inputMoney) < 1) {
+			throw new Error(MESSAGE.ERROR.NOT_NUMBER)
+		}
```

Comment:
```suggestion
		if (Number(inputMoney) < 1) {
			throw new Error(MESSAGE.ERROR.NOT_NUMBER)
		}
```
이 메서드의 목적이 결론적으로는 1보다 작은지만 체크하는 것 같아서 조건을 하나만 적어도 좋을 것 같다고 생각하는데 어떻게 생각하시나요??

Reply:
꼼꼼히 봐주셔서 감사합니다!
맞습니다 저도 다시 보면서 저 부분은 없어도 되겠더라구요!